roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -183,6 +183,13 @@
     msg: "kube_pods_subnet cannot be the same network segment as kube_service_addresses"
   run_once: yes
 
+- name: "Check that IP range is enough for the nodes"
+  assert:
+    that:
+      - 2 ** (kube_network_node_prefix - kube_pods_subnet | ipaddr('prefix')) >= groups['k8s_cluster'] | length
```

Comment:
The additional check follows the samples of
https://github.com/kubernetes-sigs/kubespray/blob/937e64d2960c98f4662c979851d9dafa7048a914/roles/kubespray-defaults/defaults/main.yaml#L192-L202

That seems good for me.